finetune_instruct_pix2pix.py

```diff
@@ -462,7 +468,7 @@ def main():
         gradient_accumulation_steps=args.gradient_accumulation_steps,
         mixed_precision=args.mixed_precision,
         log_with=args.report_to,
-        logging_dir=logging_dir,
+        project_dir=logging_dir,
```

Comment:
Could we incorporate these changes like so?

https://github.com/huggingface/diffusers/blob/fe5911bf3d794e71d6cd64da9da0e90cf735d6ca/examples/dreambooth/train_dreambooth_lora.py#L679C1-L686C6

Reply:
Of course, we can, but is that needed in the scope of this example? If so lmk and I'll add that one too. I would have to test it before pushing and I don't have time to spin up a testing environment for this one. But again, if is necessary and I'll create time then will do it.

Reply:
Understand.